v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+    + Body
+
++ Response 422
+Returned if weekday is not in range 1-7, weekday already exists, time is invalid, weekday or times are missing, time is overlapping or an error occurred.
+    + Attributes (Api Error)
+
+### Delete a Times [DELETE /owner/parking-spaces/{parking_space_id}/periods/{period_id}/times/{times_id}]
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + period_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the period id
+    + times_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Times)
+
++ Response 404
+Returned if parking space, period or time was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Body
+
+# Data Structures
+
+## Parking space (object)
++ `id`: `005c4826-4e28-11e3-a675-d43d7eece53d` (string)
++ `commission`: `33.33` (number, in percent)
++ `status`: `active` (string)
++ `currency`: `EUR` (string)
++ `code`: `ASDF12` (string)
++ `type`: `offstreet` (string)
++ `price` (Owner Price)
++ `options` (array) - describes which options are available at the location and can contains each of the following elements. The list can be **empty**.
+    + (enum)
+      + `excess length` - the parking space can be used by cars with a small trailer
+      + `indoor` - the parking space has a roof or can be an underground garagethe parking space has a roof or can be an underground garage
+      + `disabled` - parking space is accessible for disabled peopleI
+      + `patio` - the parking space is located at an inner courtyard the parking space is located at an inner courtyard
+      + `barrier` - parking space has a barrier and user needs a smartphone to open it parking space has a barrier and user needs a smartphone to open it
+      + `charging station` - parking space with an electric vehicle charging station
+      + `key_necessary` - parking space can only be accessed with a keyparking space can only be accessed with a key
+      + `long_term_booking` - parking space is bookable for longer time period
+      + `surveillance` - parking space has surveillance
+      + `valet_service` - parking space contains valet serviceparking space contains valet service
++ `post_booking_info`: `Hallo, bitte lassen Sie mich allein.`
++ `earliest_start`: `-15` (number) - bookable_range_start in minutes
++ `latest_end`: `43200` (number) - bookable_range_end in minutes (should be max 183 days)
++ `minimum_duration`: 30 - booking_duration_min
++ `maximum_duration`: 43200 - booking_duration_max (should be max 183 days)
+
+## Description (object)
++ `id`: `005c4826-4e28-11e3-a675-d43d7eece53d` (string)
++ `locale`: `de_DE` (string)
```

Comment:
use Include syntax here (just as I did with `+ Include Currency` everywhere), make a new data structure called `Locale` that uses Member syntax for that

Reply:
Done